wondrous-app/components/Common/Image/SafeImage.tsx

```diff
@@ -116,3 +117,5 @@ export function SafeImage(safeImageArgs: SafeImageArgs) {
 
   return null;
 }
+
+export default memo(SafeImage);
```

Comment:
I've added memo for image component also

Reply:
> I've added memo for image component also

great. How about adding control over the comparison and provide a custom comparison function? Regarding previous comment related to the inline styles, we can compare fields in the `prevProps.style` and `nextProps.style` tp prevent re-render.

Reply:
Yeah, it's definitely a good idea, will do, but we still have rerendering on OrgProfilePicture level, or do you think we can leave it as is?

Reply:
We can leave it as is

Reply:
I have added custom props comparison for SafeImage and rolled back inline styles

Reply:
But anyway inline styles looks not good